content/src/fieldConfig/config.json

```diff
@@ -46,33 +46,50 @@
   },
   "navigationDefaults": {
     "navBarUnnamedDbaBusinessText": "Unnamed DBA",
+    "navigationQuickLinks": {
```

Comment:
pebble / sand: given the complexity of the NAV bar, we should extract the content in the config.Json to another file. We are actively trying to not add to this file, because the end goal is to delete it completely. Given the amount of time you have taken on this large ticket, perhaps it makes sense to create a chore to re-factor this. I also think given the different variations of the Nav Bar, we should probably add a preview so content can view what the changes look like in the CMS.

Reply:
That makes sense, I would say given the amount of time on the ticket and the amount of work in doing the preview as well it would probably make more sense as a refactor ticket and I can try and pick it up on Friday